packages/teleport/src/lib/term/enums.js

```diff
@@ -34,4 +34,5 @@ export const TermEventEnum = {
 
 export const StatusCodeEnum = {
```

Comment:
Might be worth linking to https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent/code and even renaming this to something like `WebsocketCloseCode` so it's clear that this is a protocol thing and not arbitrary values.

As a side note, I'm not a huge fan of naming every enum `FooEnum`. We don't name every const `FooConst` or every class `FooClass`, so it feels odd to do that only for enumerations. But it appears there's precedent here so I'll leave that part up to you.

Reply:
I believe we're doing that because this is a Javascript file which doesn't support enums. I can convert it to Typescript and remove the `Enum` suffix

Reply:
416841ec54dfedf06b52c8dc645fb7f4b105f355